src/components/pages/dashboard/panels/analytics/analyticsPanel.js

```diff
@@ -121,6 +122,10 @@ export class AnalyticsPanel extends Component {
               }
             </div>
           </div>
+          {
+            (!showOverlay && !topAlerts.length && !Object.keys(alertsPerDeviceId).length)
```

Comment:
can topAlerts be undefined ? Need a check for that ?
Same for Object.keys(alertsPerDeviceId)

Reply:
No, they are always defined in the dashboard.